### DISCUSSION TITLE
Windows on ARM support

### DISCUSSION THREAD

**Discussion by peterjuras** [espanso/espanso#1973] [Feature requests and Ideas]
Hi!

I want to re-start a feature request from a previous issue and [discussion](https://github.com/espanso/espanso/discussions/1180):

Please consider offering an ARM build for espanso for Windows. [The new Surfaces](https://blogs.microsoft.com/blog/2024/05/20/introducing-copilot-pcs/) (along other new devices) launched with a new ARM based chipset, and the consumer models no longer feature x64 architecture variants. I preordered and will be switching and will not be able to use espanso anymore.

Alternatively, does anyone know an alternative to espanso that works on Windows on ARM?

EDIT Jan-21 2025:
For what it's worth: I'm using espanso on a Surface Laptop 7 (with a Snapdragon X ELite ARM chip) and it's fine so far. I get slowdowns sometimes but it's similar to previous experiences on non-ARM Laptops. I'm also very happy with battery life so it appears that Espanso at least does not have a tremndously negative impact on that.

**Comment by cognociente** [comment]
I strongly second this request - love using Espanso and would hate to be without it on Windows arm64 and have just bought a surface laptop

**Comment by AucaCoyan** [comment]
Hi! Sadly we aren't capable of develop a version in Windows arm64, because we don't have any on the team that have one of those pc.
Having the computer (mac silicon arch, linux based with wayland and X11 or windows) it's a requirement to compile and try to understand errors of compilation.
Because we are a small team of ~3 people, we don't target windows arm64 for the foreseeable future. Sorry!
